test/suites/network.sh

```diff
@@ -15,6 +18,14 @@ test_network() {
   lxc network set lxdt$$ ipv6.dhcp.stateful true
   lxc network delete lxdt$$
 
+  # edit network description
+  config_file=$(mktemp)
+  lxc network create lxdt$$
+  lxc network show lxdt$$ | sed 's/^description:/description: foo/' > "$config_file"
+  lxc network edit lxdt$$ < "$config_file"
+  lxc network show lxdt$$ | grep -q 'description: foo'
+  lxc network delete lxdt$$
```

Comment:
AFAICS `lxc network [set|unset]` are meant to modify the network configuration, not any top-level network attribute (like name, description or type).

It's not currently possible to change the description through them.